Makefile

```diff
@@ -0,0 +1,23 @@
+LANG = python3
```

Comment:
You shouldn't make backend changes in a frontend PR. Confirm with Cris that this file is good to go, in the future try to separate backend and frontend changes into their own PRs please